app/javascript/components/Payouts/index.tsx

```diff
@@ -839,25 +840,18 @@ const Payouts = ({
                         )}
                       </div>
                     </div>
-                    <div>
-                      <p>Amount</p>
-                      <div>${formatPriceCentsWithoutCurrencySymbol("usd", instantPayoutAmountCents)}</div>
-                    </div>
-                    <div>
-                      <p>Instant payout fee ({INSTANT_PAYOUT_FEE_PERCENTAGE * 100}%)</p>
-                      <div>
-                        -$
-                        {formatPriceCentsWithoutCurrencySymbol("usd", instantPayoutFee)}
-                      </div>
-                    </div>
+                    <PayoutLineItem title="Amount" price={instantPayoutAmountCents} />
+                    <PayoutLineItem
+                      title={`Instant payout fee (${INSTANT_PAYOUT_FEE_PERCENTAGE * 100}%)`}
+                      price={-instantPayoutFee}
+                    />
                   </div>
-                  <footer>
-                    <p>
-                      <strong>You'll receive</strong>
-                    </p>
-                    <div>
-                      ${formatPriceCentsWithoutCurrencySymbol("usd", instantPayoutAmountCents - instantPayoutFee)}
-                    </div>
+                  <footer className="grid gap-4 p-4">
+                    <PayoutLineItem
+                      title="You'll receive"
+                      price={instantPayoutAmountCents - instantPayoutFee}
+                      className="*:text-lg"
```

Comment:
```suggestion
                      className="text-lg"
```
Should be enough, right?

Reply:
Got the same effect by explicitly inheriting font size and line height on the h4; it's more verbose but I think it's more obvious that we should remove it once it's no longer necessary